src/main/java/de/unijena/cheminf/mortar/model/util/CollectionUtil.java

```diff
@@ -48,78 +48,69 @@ private CollectionUtil() {
     //</editor-fold>
     //
     //<editor-fold desc="public static methods" defaultstate="collapsed">
-    //TODO: check parameters, change params to enum values (and bools?), throw exceptions!
     /**
      * Sorts given list by property and sort type.
      *
      * @param aList List
      * @param aProperty String
-     * @param aSortType String
+     * @param ascending boolean whether sort type is ascending or descending
      */
-    public static void sortGivenFragmentListByPropertyAndSortType(List<? extends MoleculeDataModel> aList, String aProperty, String aSortType) {
-        Collections.sort(aList, (m1, m2) -> {
+    public static void sortGivenFragmentListByPropertyAndSortType(List<? extends MoleculeDataModel> aList, String aProperty, boolean ascending) {
```

Comment:
It could be surrounded by a try-catch-block to catch the ClassCastExcpetion but what should happen then? 
The class could be checked before parsing and if it does not match 0 can be returned to not sort the list.
What do you expect here?

Reply:
Returning 0 in that case would be fine, I guess. Consistent with the issue below.

Reply:
Or throw IllegalArgumentException since you also do that below if the property cannot be parsed into an enum constant?

Reply:
Then I will throw an IllegalArgumentExcpetion because returning 0 could be confusing.

Reply:
Alright, thank you 👍